client/src/pages/SearchPage.js

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
+import VideoLists from "../components/VideoLists";
+const SearchPage = ({ allVideos, getAllVideos, order }) => {
+  const [search, setSearch] = useState("");
+  const [videoSearch, setVideoSearch] = useState([]);
```

Comment:
It's not super clear from the naming what `videoSearch` is and how it's different from `search` - can you think of clearer names to make their differences and roles clear?

Or is there a way you could even get rid of one of the states, and not need them both?